import-mailbox-to-gmail.spec

```diff
@@ -0,0 +1,21 @@
+# -*- mode: python -*-
```

Comment:
Can you add some comment(s) to the file? All of these lines just seem arbitrary. :) At least mention it's for pyinstaller.